modules/AOR_Reports/AOR_Report.php

```diff
@@ -1324,8 +1324,40 @@ function build_report_query_where($query = array()){
                         $value = "{$value} OR {$field} IS NULL";
                     }
 
-                    if (!$where_set) $query['where'][] = ($tiltLogicOp ? '' : ($condition->logic_op ? $condition->logic_op . ' ': 'AND ')) . $field . ' ' . $app_list_strings['aor_sql_operator_list'][$condition->operator] . ' ' . $value;
+                    if ($condition->value_type == "Period") {
 
+                        $date = date('Y-m-d H:i:s');
+                        if (array_key_exists($condition->value, $app_list_strings['date_time_period_list'])) {
+                            $params = $condition->value;
+                        } else {
+                            $params = base64_decode($condition->value);
+                        }
+                        $value = '"' . getPeriodDate($params)->format('Y-m-d H:i:s') . '"';
+
+                        switch ($app_list_strings['aor_sql_operator_list'][$condition->operator]) {
+                            case "=":
+                                if (!$where_set) $query['where'][] = $field . ' BETWEEN ' . $value .  ' AND ' . '"' . $date . '"';
```

Comment:
I don't think you need to check if for !where_set on every line, you should check this before you enter this if/else